### DISCUSSION TITLE
Wolf Wärmepumpe

### DISCUSSION THREAD

**Discussion by seirpuk** [john30/ebusd#950] [General]
Einstellungen Homeassistant MQTT für Wolf Heizgeräte und Wärmepumpen

**Comment by mozo178** [comment]
Hi, seirpuk,

I have a half solution...
If you use this: https://github.com/john30/ebusd-configuration/tree/master/ebusd-2.x.x/de, but change the  whole 08.csv to this
https://forum.fhem.de/index.php?topic=89836.0 and change the r to r1 in that cases when there is a description, some datas are working:

![image](https://github.com/john30/ebusd/assets/40774907/02ec018f-8267-46be-8f80-34b21d77f5c0)

I use daniel's ebus adapter (https://github.com/danielkucera/esp8266-arduino-ebus), and the ebusd addon for home assistant. unfortunetaly the HA Mqtt autorecognize only the broadcast datas:
![image](https://github.com/john30/ebusd/assets/40774907/c927fe8b-7397-4d38-b5c7-761b59fb77c4)

So I will try to adapt the datapoints via nodered.

And unfortantely I cannot set setpoint values. :(

I have a BWL-1S with an AM module.

**Reply by asmaps** [reply]
Hi, thanks for that link to the FHEM forum, got me a huge step further. Got a Wolf CHA-10. Unfortunately it only works partially, did you pursue this further and have a better configuration/csv? Would you mind sharing your setup?

**Reply by puni2k** [reply]
Hi there,

if you're still interested i pretty much completed the config file for my CHA-10. Only reading status values though, no configuration parameters and also no writing.
https://github.com/puni2k/ebusd-configuration-wolf

**Comment by puni2k** [comment]
For anybody interested, i now also managed to create a config file for the Wolf MM-2
https://github.com/puni2k/ebusd-configuration-wolf/blob/main/config_mm.csv
